content_scripts/hints.js

```diff
@@ -1,5 +1,5 @@
 var Hints = (function(mode) {
-    var self = $.extend({name: "Hints", eventListeners: {}}, mode);
+    var self = $.extend({name: "Hints", eventListeners: {}, lastCreateAttrs: {}}, mode);
```

Comment:
Will do, and it looks like you use the underscore prefex, I'll update to match other private variables.